theme.json

```diff
@@ -0,0 +1,413 @@
+{
+  "$schema": "https://schemas.wp.org/trunk/theme.json",
+  "version": 2,
+  "settings": {
+    "appearanceTools": true,
+    "useRootPaddingAwareAlignments": true,
+    "border": {
+      "color": false,
+      "radius": false,
+      "style": false,
+      "width": false
+    },
+    "color": {
+      "text": false,
+      "background": false,
+      "link": false,
+      "custom": false,
+      "defaultPalette": false,
+      "defaultGradients": false,
+      "defaultDuotone": false,
+      "customGradient": false,
+      "customDuotone": false,
+      "palette": [
+        {
+          "slug": "text",
+          "color": "var(--wp--custom--text--secondary)",
+          "name": "text"
+        },
+        {
+          "slug": "heading",
+          "color": "var(--wp--custom--text--primary)",
+          "name": "heading"
+        },
+        {
+          "slug": "background",
+          "color": "var(--wp--custom--background--primary)",
+          "name": "background"
+        }
+      ],
+      "gradients": []
+    },
+    "custom": {
+      "body": {
+        "1": {
+          "font-family": "Aktiv Grotesk VF",
+          "font-weight": 400,
+          "line-height": "140%",
+          "font-size": "16px"
```

Comment:
I always recommend using rem instad of px assuming a base rem of 16px. This makes type scaling more consistent across browsers especially for users that are increasing the type size. WCAG standards require the application to work when the type is scaled 400% so using rems everywhere for spacing as well as type sizes makes this a lot more reliable.

Reply:
Good point, I'll change that up. I suspect because I picked trunk instead of develop from the valet theme, there's items that are old and have likely been changed up.

I'll change 16px to 1rem and 18px to 1.125rem.